release_build_files/readme.md

```diff
@@ -576,6 +576,10 @@ code.
     -   General (Android): Use non-conflicting file names for embedded resources
         in Android builds. This fixes segfault crashes on old Android devices
         (Android 5 and below).
+    -   Database: Fixed a potential crash that can occur as a result of a race
```

Comment:
I beleive that 7.1.1 is already out with different release notes. Should this be in the next releasE?